lib/tasks_private/openstack.rake

```diff
@@ -34,4 +34,65 @@ namespace :vcr do
     Rake::Task['vcr:spec:run'].invoke
     Rake::Task['vcr:credentials:obfuscate'].invoke
   end
+
+  task :rerecord_v2 => [] do
+    require 'fog/openstack'
+
+    host = YAML.load_file("config/secrets.yml").dig("test", "openstack", "hostname")
+    port = YAML.load_file("config/secrets.yml").dig("test", "openstack", "port")
+    username = YAML.load_file("config/secrets.yml").dig("test", "openstack", "userid")
+    password = YAML.load_file("config/secrets.yml").dig("test", "openstack", "password")
```

Comment:
This should already be handled in `spec/spec_helper.rb` here https://github.com/ManageIQ/manageiq-providers-openstack/blob/master/spec/spec_helper.rb#L15-L22
You can access it like `Rails.application.secrets.openstack.hostname`

Reply:
Just noticed this is in a rake task not a spec test, ignore ^ for now it'll be useful later

Reply:
More efficient to read and parse the file once then access the hash
```suggestion
    host, port, username, password = YAML.load_file("config/secrets.yml").dig("test", "openstack").values_at("hostname", "port", "username", "password")
```